model/task_lifecycle.go

```diff
@@ -123,6 +121,11 @@ func SetActiveState(caller string, active bool, tasks ...task.Task) error {
 		}
 	}
 
+	for _, t := range tasksToActivate {
```

Comment:
display tasks werent correctly updated before because we never deactivated the execution tasks before we updated them